CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 * UI tests replacement with RTL/Jest for component `RulesEditor`. Refs UICIRC-827.
 * UI tests replacement with RTL/Jest for `Rules-hint`. Refs UICIRC-831.
 * Add metadata information to view of Staff Slips (Settings > Circulation > Staff Slips). Refs UICIRC-855.
+* Implement General information accordion. Refs UICIRC-909
```

Comment:
Could you please update CHANGELOG.md (example below)?
```
## IN PROGRESS

* Add metadata information to view of Staff Slips (Settings > Circulation > Staff Slips). Refs UICIRC-855.
* Implement General information accordion. Refs UICIRC-909
* Implement Template content accordion. Refs UICIRC-910
 
## [8.0.0](https://github.com/folio-org/ui-circulation/tree/v8.0.0) (2023-02-22)
[Full Changelog](https://github.com/folio-org/ui-circulation/compare/v7.2.1...v8.0.0)

* Make consistent view of delete functionality of "Fixed due date schedules". Refs UICIRC-874.
...
* Add metadata information to view of Staff Slips (Settings > Circulation > Staff Slips). Refs UICIRC-855. // remove
```

Reply:
Thanks for catching it @Dmitriy-Litvinenko 
Corrected it